### ISSUE TITLE
use_complex_stft_discriminator = False

### ISSUE THREAD

**Issue by inspirit** [lucidrains/audiolm-pytorch#64]
Hi Phil,

after setting `use_complex_stft_discriminator = False` i can see that discr losses are zero or very small:
`soundstream total loss: 177.001, soundstream recon loss: 1.143 | discr (scale 1) loss: 0.000 | discr (scale 0.5) loss: 0.000 | discr (scale 0.25) loss: 0.000`

also total loss seems to be too high

when setting `use_complex_stft_discriminator = True`

losses look more normal to me:
` soundstream total loss: 12.153, soundstream recon loss: 0.007 | discr (scale 1) loss: 1.184 | discr (scale 0.5) loss: 4.218 | discr (scale 0.25) loss: 2.376`

**Comment by lucidrains** [comment]
@inspirit hey Eugene, think i found one issue (although there may be more) would you like to try again?

**Comment by inspirit** [comment]
restarted seems fine at start, before it quickly dropped to zeros in first 100-200 steps.
i also noticed that with current default LR=3e-4 its quite unstable and explodes at some point, maybe it make sense to reduce default LR?

**Comment by lucidrains** [comment]
@inspirit thank you for verifying! fantastic news

i'll lower the default learning rate to `2e-4` as well
